src/main/java/jrm/dao/UserRepository.java

```diff
@@ -3,14 +3,25 @@
 
 import jrm.exeptions.UserNotFoundException;
 import jrm.user.User;
+import org.springframework.data.jpa.repository.Modifying;
+import org.springframework.data.jpa.repository.Query;
 import org.springframework.data.repository.CrudRepository;
 import org.springframework.stereotype.Repository;
 
+import javax.transaction.Transactional;
+import java.sql.Timestamp;
 import java.util.Optional;
 
 @Repository
 public interface UserRepository extends CrudRepository<User, Long> {
 
+    // update creation date
+    @Transactional
+    @Modifying
+    @Query(value = "UPDATE jrm_user SET  created_on = ?1 WHERE id = ?2", nativeQuery = true)
+    void updateUserCreatedDate(Timestamp created_on, Long id);
```

Comment:
Java naming conventions are broken here, should be createdOn. You can map to sql variable using @Param annotation if you wanna to use "created_on" in the native query: `@Param("created_on") Long createdOn`
https://docs.oracle.com/javase/tutorial/java/nutsandbolts/variables.html

Reply:
Fixed.